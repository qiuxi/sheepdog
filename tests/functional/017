#!/bin/bash

# Test vdi tree functionality

. ./common

for i in `seq 0 5`; do
    _start_sheep $i
done

_wait_for_sheep 6

_cluster_format -c 1

_vdi_create test 4G
$QEMU_IMG snapshot -c tag1 sheepdog:test
$QEMU_IMG snapshot -c tag2 sheepdog:test
$QEMU_IMG snapshot -c tag3 sheepdog:test

_vdi_create test2 4G
$QEMU_IMG snapshot -c tag1 sheepdog:test2
$QEMU_IMG snapshot -c tag2 sheepdog:test2
$QEMU_IO -c "write 0 512" sheepdog:test2:1 | _filter_qemu_io
$QEMU_IMG snapshot -c tag3 sheepdog:test2

$DOG vdi tree | _filter_short_date
