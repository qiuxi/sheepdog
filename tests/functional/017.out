QA output created by 017
using backend plain store
wrote 512/512 bytes at offset 0
512 bytes, X ops; XX:XX:XX.X (XXX YYY/sec and XXX ops/sec)
test---[DATE]---[DATE]---[DATE]---(you are here)
test2---[DATE]-+-[DATE]
                           `-[DATE]---(you are here)
